Class/ClassHelper.psm1

```diff
@@ -0,0 +1,44 @@
+function LogOutput {         
+    [CmdletBinding()]
+    param($msg)
+    $timestamp = (Get-Date).ToUniversalTime()
+    $output = "$timestamp [INFO]:: $msg"    
+    if ($VerbosePreference -ne "SilentlyContinue") {
```

Comment:
Write-Verbose is executed just when VerbosePreference is set to silentlycontinue, hence you don't need the if statement.